recipe/meta.yaml

```diff
@@ -50,9 +49,8 @@ outputs:
 
   - name: uvicorn-standard
     build:
-      skip: true  # [py<38]
-      # watchfiles is missing on s390x and win-64
-      skip: true  # [s390x or win]
+      # s390x missing watchfiles
+      skip: true  # [py<38 or py>311 or s390x]
```

Comment:
If only `watchfiles` is missing py312 artifacts, then it would be nice to update it to the latest version 0.24.0 (the current version is 0.21.0)